Sources/GoogleAI/GenerateContentResponse.swift

```diff
@@ -135,16 +152,28 @@ extension CandidateResponse: Decodable {
 
 /// A collection of source attributions for a piece of content.
 public struct CitationMetadata: Decodable {
+
+  /// A list of individual cited sources and the parts of the content to which they apply.
   public let citationSources: [Citation]
 }
 
+/// A struct describing a source attribution.
 public struct Citation: Decodable {
+
+  /// The beginning of a sequence in a model response, inclusive, that derives from a cited source.
```

Comment:
My preference would be:

The inclusive beginning of a sequence in a model response that derives from a cited source.

Reply:
Done.